src/Uno.UWP/UI/Composition/CompositionPropertySet.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
```

Comment:
```suggestion
#nullable enable
﻿
﻿using System;
```

Reply:
> How does the control behave without the composition scaling ?

It lacks the nice animations, but it does work properly (e.g. you can see/change rating)